index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="container">
+       <h1 class="name">John Doe</h1>
```

Comment:
As this is a logo and it can be used for going to the home page in the future that's why I'll recommend you to please replace `<h1>` tag with `<a>` tag. 👍🏻